syntax = "proto3";

package sensor;

service SensorService {
  rpc SendData (SensorData) returns (Response);
}

message SensorData {
  string nombre_sensor = 1;
  string fecha_registro = 2;
  string tipo_dato = 3;
  float valor = 4;
}

message Response {
  string status = 1;
}